src/RequestHandler.php

```diff
@@ -118,6 +118,10 @@ public function handle(Request $request) {
       $frontend_request = $this->requestGenerator->getFrontendRequest($request);
       list($frontend_response, $backend_response) = $this->responseFetcher->fetchResponses($frontend_request, $backend_request);
 
+      if ($this->requestGenerator->isFrontendPage($request)) {
```

Comment:
Let's not extend the logic here. This would require interface changes.
setFrontendPages() is documented to allow more pages like layout--default - it's not documented to be about "frontend-only" paths. Also, we still need/use the backend response for simple things like setting metatags and breadcrumbs with the backend values.
The content replace will be simply skipped if https://github.com/drunomics/lupus-frontproxy/blob/1.x/src/ResponseMerger/NuxtResponseMerger.php#L44 does not find the right element to replace.

Reply:
thus, simply remove that hunk

Reply:
karim said that these pages should not be touched by frontend proxy that is why I prevented it

Reply:
just also tested this -> getting hydration errors again if I remove that... NuxtResponseMerger still mangles it...